votes/vote-20220125-revertn-aes-noasm.txt

```diff
@@ -0,0 +1,21 @@
+Topic: OTC approves the OMC request for an exception to the stable release
```

Comment:
This is mistakenly added to the PR.

Reply:
Hmmmm, not sure where that is coming from.

Reply:
I'm only seeing two commits before I get to the master branch.  That's the one with the vote in it.
When I push, the third magically appears here.

Reply:
Ah, the commit that added the vote was pushed to github.com mirror repo but not to the GHE primary repo.

Reply:
So the 2592430 needs to be pushed to master on github.openssl.org and things should fix themselves.

Reply:
Thanks.  I'd made a muddle of my clone of this repo and the primary repo was wrong...